### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by lstables** [mstnorris/laravel-network#1]
Hi dude,

Question for ya!

How does this https://github.com/mstnorris/laravel-network/blob/master/app/Mailers/AppMailer.php send email do you still need to active SMTP creds in .env file? and can you have many methods as long as they use the "deliver()" method too?

**Comment by lstables** [comment]
Having played with it yes it does, so closed this.

**Comment by mstnorris** [comment]
Hey,

Guzzle is already in the composer.json file, and you need to set up Mandrill i.e. add `MAIL_DRIVER=mandrill` and a `MANDRILL_KEY=whatever` to your .env file. Then, in your config/services.php file make sure you have:

'mandrill' => [
    'secret' => env('MANDRILL_KEY'),
],

I haven’t fully implemented it yet, will try to work on it over the next couple of days but I’m starting a big project now so all my time is going on that.

Feel free to fork the project though and if you have any questions then just shout.

-M

**Comment by lstables** [comment]
Yes I realised that after I sent this (slaps head) tried it and it just works then I did `$mail->sendQuestionsToAdmin($user);` with the AppMailer type hinted on the method and just works. But thanks for the heads up. Trying to learn a better different approach to building a similar app, not so much a social network as such, but more like a forum where people can post questions and answers. 

Ideally needed a much better way to send mail, and now I can thanks to you :)
